### ISSUE TITLE
Hogwarts Legacy DLSS 3 FG Option stays greyed out

### ISSUE THREAD

**Issue by MelvinSmiley** [Nukem9/dlssg-to-fsr3#409]
No matter what I try, the DLSS 3 FG Option does not become available for some reason. FSR3 mod works fine with Remnant 2 on my system with 3080 12 gb, so I don't really get it. HAGS on, VSync off, DLSS Quality, no matter what I do, the FG option stays greyed out for some reason. Strange. Using 0,9 version of the mod. I have the fully updated Hogwarts Legacy steam version.

**Comment by LeBlackMeow** [comment]
The two DLL need to be placed next to the executable. Are you using the universal one or the regular with the registry check?

**Comment by Vladzor87do** [comment]
Use wiki
https://github.com/Nukem9/dlssg-to-fsr3/wiki/Hogwarts-Legacy

exe at the game folder is just a shortcut exe. Real exe is in Hogwarts Legacy\Phoenix\Binaries\Win64

another thing is - you need to activate HAGS in Windows

**Comment by MelvinSmiley** [comment]
Yes I know that real exe is in Hogwarts Legacy\Phoenix\Binaries\Win64. HAGS is also turned on. Still doesn't work for some reason. Tried updating my Nvidia driver to the newest version, still no luck. Frame generation menu entry stays greyed out no matter what I do.

**Comment by Aidorin0** [comment]
> Yes I know that real exe is in Hogwarts Legacy\Phoenix\Binaries\Win64. HAGS is also turned on. Tried updating my Nvidia driver to the newest version, the nvngx.dll method and the Universal version.dll method like shown in the tutorial video so far, also deleted the Hogwarts Legacy configuration file, still no luck, FG option just stays greyed out. Will give up at this point I think, it just doesn't work.

That's weird. It worked fine in my case with v0.9 and version.dll. Maybe it's because that _fully updated Hogwarts Legacy steam version_. I didn't use that.

**Comment by LeBlackMeow** [comment]
[MelvinSmiley](https://github.com/MelvinSmiley), it is the jack sparrow edition ?

**Comment by MelvinSmiley** [comment]
> [MelvinSmiley](https://github.com/MelvinSmiley), it is the jack sparrow edition ?

No, bought the game on steam.

**Comment by LeBlackMeow** [comment]
> > [MelvinSmiley](https://github.com/MelvinSmiley), it is the jack sparrow edition ?
> 
> No, bought the game on steam.

Then, according to the [compatibility](https://github.com/Nukem9/dlssg-to-fsr3/wiki/Game-Compatibility-List) list, it should work without issues.

**Comment by MelvinSmiley** [comment]
> > > [MelvinSmiley](https://github.com/MelvinSmiley), it is the jack sparrow edition ?
> > 
> > 
> > No, bought the game on steam.
> 
> Then, according to the [compatibility](https://github.com/Nukem9/dlssg-to-fsr3/wiki/Game-Compatibility-List) list, it should work without issues.

I took a look at the compatibility list before trying the game. It's just wrong in my case.

**Comment by Neegger** [comment]
Same here, but not working even after reinstall
